app/controllers/dashboard_controller.rb

```diff
@@ -60,6 +61,22 @@ def autoyast
     end
   end
 
+  # Read the keyboard layout set by Yast during installation.
+  #
+  # @return [String] the keyboard set from Yast, or 'english-us' as
+  # default
+  def read_keyboard_layout(keyboard_config_file: "/var/lib/misc/keyboard")
+    default_keyboard_layout = "english-us"
+    yast_keyboard_key = "YAST_KEYBOARD"
```

Comment:
I would make both `yast_keyboard_key` and `default_keyboard_layout` constants and move them at the top of the class definition.